doc/starlight/tsconfig.json

```diff
@@ -1,5 +1,12 @@
 {
   "extends": "astro/tsconfigs/strict",
   "include": [".astro/types.d.ts", "**/*"],
-  "exclude": ["dist"]
+  "exclude": ["dist"],
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+        "@components/*": ["src/components/*"],
+    },
+    "types": ["astro/client", "astro/astro-jsx"]
+  }
```

Comment:
what does this change do?

Reply:
Is there a way to add this as a comment to the config? (JSON doesn't do comments, does it?)

Do you think the usage of `@components/<your_import>` should be explained in the Guide-Guide?

Reply:
Sure, I can add a section to the guide about that

Reply:
Sure, I can add a section to the guide about that

Reply:
done